### ISSUE TITLE
smart indent breakage for mail and gitcommit filetypes

### ISSUE THREAD

**Issue by fourjay** [sheerun/vim-polyglot#552]
**Does this bug happen when you install plugin without vim-polyglot?**

No

**Describe the bug:**

Started recently (not sure exact time frame)
Basic tab settings (all space)
set softtabstop=4
set shiftwidth=4
set expandtab

So far two files with auto-population behavior overwrite one or more defaults

Open a git commit, and defaults are overridden
verbose sw?
Last set from ~/.dotfiles/vim/bundle/vim-polyglot/ftdetect/polyglot.vim line 2129
gitcommit is prepopulated

Sending a mail message (mutt) and expandtab is overwritten
Also prepopulated (with email header information.

The idea is great, but this seems too aggressive.

**Comment by sheerun** [comment]
Could you provide examples of mutt and gitcommit contents? It's much easier to debug this way

**Comment by sheerun** [comment]
ping

**Comment by fourjay** [comment]
I apologize.
I took the FT overides out, and tried to replicate, and could not easily do so.
I suspect the issue comes from both git and mutt's auto filling in a random way.
I use mutt to respond to emails, which (typically) copies the text of the original message with quote marks.
Which leads me to suspect that some message are seen with odd indents.
When I first noticed this, ``sw`` was set to 9, which potentially makes sense for a normal shift width plus quote character.
I'll need to look more at gitcommits, as I noticed this on a satellite machine, not my main workstation, and this might be related to differing behavior of git on the two machines

That said, things are behaving as expected now.
I'll update the ticket (if) when I see the odd output with the text
